Assets/Scripts/Models/World.cs

```diff
@@ -216,13 +219,16 @@ public void RandomizeTiles()
         {
             for (int y = 0; y < Height; y++)
             {
-                if (UnityEngine.Random.Range(0, 2) == 0)
-                {
-                    tiles[x, y].Type = TileType.Empty;
-                }
-                else
+                for (int z = 0; z < Depth; z++)
```

Comment:
It seems that this method ( `RandomizeTiles()` ) is not used anywhere.
Should we delete it?